### ISSUE TITLE
%% mistaken for placeholder in non-replacing call

### ISSUE THREAD

**Issue by Rarst** [WordPress-Coding-Standards/WordPress-Coding-Standards#681]
```
FILE: ...p-content\plugins\wordpress-seo\inc\class-wpseo-replace-vars.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 109 | ERROR | [x] Multiple placeholders should be ordered. Expected
     |       |     '%1$%, %2$%', but got %%,%%.
     |       |     (WordPress.WP.I18n.UnorderedPlaceholders)
----------------------------------------------------------------------
```

In:

```php
trigger_error( __( 'A replacement variable can not start with "%%cf_" or "%%ct_" as these are reserved for the WPSEO standard variable variables for custom fields and custom taxonomies. Try making your variable name unique.', 'wordpress-seo' ), E_USER_WARNING );
```
https://github.com/Yoast/wordpress-seo/blob/ff4fce5db9b81723a40763502a05742041f3c878/inc/class-wpseo-replace-vars.php#L109

Not a replacing call, neither does `%%` forms a valid replace placeholder (I think).

**Comment by jrfnl** [comment]
> Not a replacing call, neither does %% forms a valid replace placeholder (I think).

Actually it does, it's the escape code for a literal `%`.

Anyways, I think I fixed it for you - would you mind testing PR #683 ?

**Comment by Rarst** [comment]
Yeah, I though it's escaping then misread the docs to think it's not.

Patch tested and works, cheers. :)
